ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -682,6 +783,15 @@ objects:
             exit 1
         fi
 
+        if [[ "${ENABLE_PROXY}" == "true" ]]; then
+          cat >> /tmp/artifacts/installer/install-config.yaml << EOF
+        proxy:
+          httpsProxy: http://admin:admin@${JOB_NAME_SAFE}-egress-proxy.${NAMESPACE}.svc:3128/
+          httpProxy: http://admin:admin@${JOB_NAME_SAFE}-egress-proxy.${NAMESPACE}.svc:3128/
+          noProxy: localhost
```

Comment:
I hear that there's a list of things you should fill in here to keep cluster-internal traffic off the cluster.  I'll see if I can get you the string for that...

Reply:
@wking you shouldn't have to set `noProxy` only if there were additional http endpoints you didn't want proxied

Reply:
ah, so in-cluster traffic won't go through the proxy even if you leave `noProxy` unset?

Reply:
right @wking I think this will help:
https://github.com/openshift/installer/blob/7d2784d43d04b2f0a784af632ff2683a377cae1a/pkg/asset/manifests/proxy.go#L122-L127

Reply:
the expectation that we set was "if your component only talks to local cloud apis, your component does not need to consume the proxy configuration".

So it's not so much that noproxy is set/unset, as it's irrelevant for those components.

If a component both talks off-cluster *and* to cloud apis, then either:
1) that component will go through the proxy for everything anyway (not great)
2) the component itself needs to figure out how to divide its transports into proxied calls and non-proxied calls (not great)
3) we need to ensure the cloud api endpoints are in the noproxy list, which is, I thought, why we were adding 169.254.169.254 to no proxy, which at least covers aws(?)

Reply:
@bparees `169.254.169.254` is in the no proxy list because bootstrap would fail if it wasn't included.  OpenStack, Azure and GCP also use that address for instance metadata.

See comment:
https://github.com/openshift/installer/blob/7d2784d43d04b2f0a784af632ff2683a377cae1a/pkg/asset/manifests/proxy.go#L106-L111

Reply:
So `noProxy: localhost` in the install-config is really a no-op, because `localhost` is already hard-coded into the no-proxy set by the installer code.

169.254.169.254 seems to be some sort of cross-cloud convention (there isn't an RFC or other spec claiming it, is there?).  For example, [here][1]'s GCP calling it out as the instance metadata server.

[1]: https://cloud.google.com/compute/docs/internal-dns#access_by_internal_DNS